Library/Formula/slimerjs.rb

```diff
@@ -25,25 +49,34 @@ def install
     bin.install_symlink libexec/"slimerjs"
   end
 
-  def caveats; <<-EOS.undent
-    You can set the environment variable SLIMERJSLAUNCHER to a installation of
-    Mozilla Firefox (or Mozilla XULRunner) to use this version with SlimerJS
-    instead of the one installed by Homebrew (this is required if built without
-    xulrunner).
-    For a standard Mozilla Firefox installation this would be:
+  def caveats
+    s = ""
 
-      export SLIMERJSLAUNCHER=/Applications/Firefox.app/Contents/MacOS/firefox
+    firefox_installation = FirefoxDependency.firefox_installation
+    unless !firefox_installation?
```

Comment:
I've pushed the current whole formula here so you can glance over it in entirety if it's helpful.